lib/msal-common/src/request/RequestParameterBuilder.ts

```diff
@@ -471,6 +476,12 @@ export class RequestParameterBuilder {
         );
         Object.keys(sanitizedEQParams).forEach((key) => {
             this.parameters.set(key, eQParams[key]);
+            if (key === "scenarioId" && this.correlationId) {
```

Comment:
Can we make `scenarioId` a first class param if we're going to take dependency on it? 2 reasons:
1. EQP should be opaque
2. Allows us to define type